File: frontend/src/types/Room.ts
export interface Room{
    roomId: number;//방id
    competitionId: number;//대회id
    name: string;//종목 이름
    password: string;//방(종목) 입장 비밀번호
    roundCount: number;//방(종목)의 라운드 수
    createdAt: string;//생성 일자
}

File: backend/src/main/java/com/mma/backend/entity/Room.java
package com.mma.backend.entity;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Room {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long roomId;

    @ManyToOne
    @JoinColumn(name = "competition_id")
    private Competition competition;

    private String name;

    private String password;

    private int roundCount;

    private LocalDateTime createdAt;
}
